Doc/tutorial/errors.rst

```diff
@@ -16,14 +16,15 @@ Syntax Errors
 
 Syntax errors, also known as parsing errors, are perhaps the most common kind of
 complaint you get while you are still learning Python::
-
-   >>> while True print('Hello world')
-  File "<stdin>", line 1
-    while True print('Hello world')
-          ^^^^^^^^^^^^^^^^^^^^^^^^^
-SyntaxError: invalid syntax. Perhaps you forgot a comma?
-
-The parser repeats the offending line and displays a row of 'arrows' pointing at
+   
+   >>> greeting = 'Hello World'
+   >>> print greeting
+     File "<stdin>", line 1
+       print greeting
+       ^^^^^^^^^^^^^^
+   SyntaxError: Missing parentheses in call to 'print'. Did you mean print(greeting)?
+
+The parser repeats the offending line and displays 'arrows' pointing at
```

Comment:
Actually now I see that this example doesn’t demonstrate what the text mentions - that the arrows point to where the error begins. Can we find an example where the arrows don’t cover the whole line?

Reply:
> Actually now I see that this example doesn’t demonstrate what the text mentions - that the arrows point to where the error begins. Can we find an example where the arrows don’t cover the whole line?

How about:

```python
>>> a[1]] = 0
  File "<stdin>", line 1
    a[1]] = 0
        ^
SyntaxError: unmatched ']'
```